### PR TITLE
update godoc for virtual endpoint body field

### PR DESCRIPTION
**Pull request by jeffy-mathew**

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/4820/merge
Commit:  
Triggered by: pull_request (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4310511586)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/4820/merge
Commit: d8588c76c1c2d5ced435dea45d04b2f53a76ecb7 
Triggered by: pull_request (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4310548982)

File: apidef/oas/middleware.go [hunk 1]
```diff
@@ -1035,7 +1035,7 @@ type VirtualEndpoint struct {
 	Name string `bson:"name" json:"name"` // required.
 	// Path is the path to js file.
 	Path string `bson:"path,omitempty" json:"path,omitempty"`
-	// Body is the js function to execute encoded in base64 format.
+	// Body represents the js function body content. It is stored in base64 format.
```
Comment (letzya) [inline apidef/oas/middleware.go:1038]: The previous description sounds clearer to me

**Review by letzya — CHANGES_REQUESTED** [review]

File: apidef/oas/middleware.go (hunk 1 above)
Comment (jeffy-mathew) [reply apidef/oas/middleware.go:1038]: This change is based on comment https://github.com/TykTechnologies/tyk-docs/pull/2330#discussion_r1121362561. 
If we decide to go without making this change, we can discard this PR and get tyk-docs PR merged.

Comment (letzya) [reply apidef/oas/middleware.go:1038]: it still needs a comma or full stop

Comment (letzya) [inline apidef/oas/middleware.go:1038]:
```suggestion
	// Body is the js function. It is stored in base64 format.
```

Comment (letzya) [reply apidef/oas/middleware.go:1038]: yes

Comment (letzya) [reply apidef/oas/middleware.go:1038]: but we need to add this `if path and body are present, body would take precedence`

File: apidef/oas/schema/x-tyk-gateway.md [hunk 2]
```diff
@@ -972,7 +972,7 @@ Name is the name of js function.
 Path is the path to js file.
 
 **Field: `body` (`string`)**
-Body is the js function to execute encoded in base64 format.
+Body represents the js function body content. It is stored in base64 format.
```
Comment (letzya) [inline apidef/oas/schema/x-tyk-gateway.md:975]:
```suggestion
Body represents the js function body content. It is stored in base64 format.
If path and body are present, body would take precedence
```

File: apidef/oas/middleware.go (hunk 1 above)
Comment (letzya) [inline apidef/oas/middleware.go:1038]:
```suggestion
	// Body represents the js function body content. It is stored in base64 format.
	// If `path` and `body` are present, `body` would take precedence.
```

**Pull request closed without merging**